src/test/java/io/sudhakar/student/controller/GenericQueryControllerTest.java

```diff
@@ -183,16 +175,16 @@ void updateStudentInternalServerError() {
         Mockito.when(genericService.updateStudent(Mockito.any(), Mockito.any()))
                 .thenReturn(serviceResponse);
 
-        ResponseEntity<?> responseEntity = genericQueryController.updateStudent(Mockito.any(), Mockito.any());
+        ResponseEntity<Void> responseEntity = genericQueryController.updateStudent(Mockito.any(), Mockito.any());
 
         assertEquals(500, responseEntity.getStatusCodeValue());
     }
 
-  /*
-  Given deleteStudent(id,queryName)
-  Then  ServiceResponse
-  Scenario SUCCESS
-  */
+    /*
+    Given deleteStudent(id,queryName)
+    Then  ServiceResponse
```

Comment:
update this comments for all the methods

Reply:
removed query name in comments for all method